evento/src/components/SearchBar/index.js

```diff
@@ -5,7 +5,12 @@ class SearchBar extends Component {
 	render() {
 		return (
 			<div className="SearchBar">
-				<input type="text" name="search" placeholder="search evento"></input>
+				<input 
+					type="text"
+					name="search"
+					placeholder="search evento"
+					onChange={this.props.updateFilter}>
```

Comment:
Do `(evt) => this.props.updateFilter(evt.target.value)` so that the callback gets the actual value instead of DOM Event object.